README.md

```diff
@@ -21,9 +21,9 @@ Let us know how you think we're doing.
 <a href="https://aka.ms/mssqljdbcsurvey"><img style="float: right;"  height="67" width="156" src="https://sqlchoice.blob.core.windows.net/sqlchoice/static/images/survey.png"></a>
 
 ## Status of Most Recent Builds
-| Azure Pipelines (Windows)       | Travis CI (Linux) |
+| Azure Pipelines (Windows) | Azure Pipelines (Linux) | Azure Pipelines (MacOS) |
```

Comment:
Please click [View file](https://github.com/microsoft/mssql-jdbc/blob/abd4a2b94184a627011f8df333f55ee7a5a7c2ee/README.md), the table seems to be broken.